frontend/src/routes/Governance/overview/Overview.tsx

```diff
@@ -385,12 +397,12 @@ function renderClusterList(
   return (
     <div style={{ paddingBottom: '10px', display: 'grid', gridTemplateColumns: '1fr auto auto auto auto', gap: 16 }}>
       {clusterList.map(({ cluster, violations }) => {
-        const key = `${cluster.metadata.name}-card`
+        const key = `${cluster?.metadata?.name}-card`
```

Comment:
Seems strange that optional chaining would be needed here. And that `cluster` is of type any. Looks like we have the types at all call sites of `renderClusterList`, so why not use them? Could we update the function like this?

```
function renderClusterList(
  clusterList: { cluster: ManagedCluster; violations: ViolationSummary }[],
  onClick: { (cluster: ManagedCluster, compliance: string): void },
  t: TFunction
) {
```

I don't follow why the `onClick` has two different signatures currently.